extensions/clippyx/src/caption.tsx

```diff
@@ -0,0 +1,136 @@
+import { ActionPanel, Action, Grid, showToast, Toast, getPreferenceValues } from "@raycast/api";
+import { useState, useEffect, useCallback } from "react";
+import { useFetch } from "@raycast/utils";
+import { exec } from "child_process";
+import { promisify } from "util";
+
+const execAsync = promisify(exec);
+
+interface Image {
+  id: string;
+  path: string;
+}
+
+interface Preferences {
+  threshold: number;
+  topK: number;
+}
```

Comment:
You can remove this part, this is automaticlly generated in `raycast-env.d.ts`

Reply:
I got this error when I tried removing it

```
src/caption.tsx:27
:30 - error TS2339: Property 'threshold' does not exist on type 'ExtensionPreferences'.

27       threshold: preferences.threshold,
                                ~~~~~~~~~

src/caption.tsx:28:26 - error TS2339: Property 'topK' does not exist on type 'ExtensionPreferences'.

28       top_k: preferences.topK,
```